homeassistant/components/ld2410_ble/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "ld2410_ble",
+  "name": "LD2410 BLE",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/ld2410_ble/",
+  "requirements": ["bluetooth-data-tools==0.3.0", "ld2410-ble==0.1.1"],
+  "dependencies": ["bluetooth"],
+  "codeowners": ["@930913"],
+  "bluetooth": [{ "local_name": "HLK-LD2410B_*" }],
```

Comment:
Is there another way to detect it without using the long name since unless it broadcasts this it won't be able to be found with passive scans

Reply:
After much trying, I can inconclusively say no: the only way it showed up as discovered was with the name. Could it just be my BT adaptor?

Reply:
It's probably just a limitation of the device itself. That's unfortunate